core/src/js/components/duels/desktop/deckbuilder/duels-deckbuilder-cards.component.ts

```diff
@@ -100,7 +100,13 @@ export const DEFAULT_CARD_HEIGHT = 221;
 						[tooltip]="'app.duels.deckbuilder.show-buckets-button-tooltip' | owTranslate"
 					></preference-toggle>
 					<div class="results">
-						<virtual-scroller class="cards-container" #scroll [items]="value.activeCards" scrollable>
+						<virtual-scroller 
+							class="cards-container" 
+							#scroll 
+							*ngIf="value.activeCards.length; else emptyState"
```

Comment:
Here I would safeguard the call to length with `value.activeCards?.length`

Reply:
Done